tests/console/docker.pm

```diff
@@ -116,6 +116,21 @@ sub run {
         assert_script_run('man -P cat docker config | grep "docker-config - Manage Docker configs"');
     }
 
+    # Try to stop container using ctrl+c
+    type_string("docker run --rm opensuse/tumbleweed sleep 20\n");
+    type_string("# Lets's press ctrl+c now ... ");
+    send_key 'ctrl-c';
+    type_string(" ... and we seem to be still in container\n");
+    save_screenshot();
+    # If echo works then ctrl-c stopped sleep
+    type_string "echo 'ctrlc_timeout' > /dev/$serialdev\n";
+    if (wait_serial 'ctrlc_timeout', 10, 1) {
+        die 'ctrl-c stopped container';
+    }
+    else {
```

Comment:
Again, optional, but you can replace `else {..}` with second wait_serial - to be sure that docker run finished
```perl
  die "Something went wrong" unless wait_serial('ctrlc_timeout', 20);
```

Reply:
Done, see [SLE12](http://pdostal-server.suse.cz/tests/808), [SLE15](http://pdostal-server.suse.cz/tests/810)